src/utils/creditCardType.js

```diff
@@ -0,0 +1,24 @@
+const CREDIT_CARDS = [
+  {
+    type: 'AM_EX',
+    regex: /^3[47][0-9]{3}/gi
+  },
+  {
+    type: 'VISA',
+    regex: /^4[0-9]{5}/gi
+  },
+  {
+    type: 'MASTER_CARD',
+    regex: /^5[1-5][0-9]{4}/gi
+  }
+];
+
+export const getCreditCardType = (cardNumber = '') => {
```

Comment:
Would change its name to **creditCardType**, without **get** prefix.
Honestly I would go for straightforward if ... else blocks for readability / performance.
So I would go for (not sure if it's worth adding regexes to consts):
```
if(cardNumber.match(/^4[0-9]{5}/gi)) {
  return CREDIT_CARD_VISA;
} else if {
 ...
}
return CREDIT_CARD_UNKNOWN
```